python/svix/api/__init__.py

```diff
@@ -0,0 +1,23 @@
+from .api import (  # noqa
+    ApplicationIn,
+    ApplicationOut,
+    DashboardAccessOut,
+    FetchOptions,
+    ListResponseApplicationOut,
+    ListResponseMessageOut,
+    MessageIn,
+    MessageOut,
+    Svix,
+    SvixOptions,
+)
+from .exceptions import (  # noqa
```

Comment:
This exports everything like this: `svix.api.ApiException`, which I don't think is what you want...